src/application.cpp

```diff
@@ -60,38 +58,71 @@ int Application::run() {
     Metrics::createDataFile(DATA_OUT);
     float cur_log, last_log = 0;
 
+    // multithreading
+    pthread_t ptid;
```

Comment:
pthread isnt a standard windows thing. std::thread looks to be an alternative, otherwise tomorrow i can try looking into getting it set up with cmake

Reply:
used std::thread now - works fine for me